modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1064,6 +1064,12 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
       }
 
       const keychain = bitcoin.HDNode.fromBase58(userPrv);
```

Comment:
Is there some case where we are providing an `xpub` as the `prv` param?

Reply:
yes I added a test case for this.